x-pack/plugins/apm/scripts/bootstrap_apm/src/run.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -u
+scriptdir="$(dirname "$0")"
+cd "$scriptdir"
+source variables.sh
+source utils/logger.sh
+
+if ! [ -x "$(command -v ecctl)" ]; then
+  echo -e "${RED}ERROR: ${NC}ecctl not found.${NC}"
+  exit 1
+fi
+
+helpFunction()
+{
+   echo ""
+   echo "Usage: $0 -v "7.17.0" -n my-deployment -r europe-west2 -p gcp-hot-warm"
+   echo -e "\t-v Version of the stack "
+   echo -e "\t-n Deployment name"
+   echo -e "\t-r Region"
+   echo -e "\t-p Hardware profile"
+   exit 1 # Exit script after printing help
```

Comment:
Should it really exit with code 1 instead of 0?

Reply:
good catch, it should be `0` since it's not a failure